src/main/java/com/akademia/educators_back/to/NewProblemTo.java

```diff
@@ -5,6 +5,9 @@
 
 import java.util.List;
 
+/**
+ * Object representing new problem
```

Comment:
jeśli mamy To - to oznacza że jest to transfer object - "Transfer object representing new Problem object for saving in database"